browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2761,7 +2761,8 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			if (typeof menubutton === 'object') {
 				L.DomUtil.addClass(menubutton.container, data.class ? data.class + ' has-colorpicker': 'has-colorpicker');
 
-				var valueNode = L.DomUtil.create('div', 'selected-color', menubutton.container);
+				var buttonElem = menubutton.container.querySelector('button');
```

Comment:
we can avoid query here, we should take reference from the menubutton object, probably it will be `menubutton.button`. if not please look into `Widget.MenuButton.ts`

Reply:
done